laravel-project/app/Http/Controllers/MemoController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Validator;
+use App\Models\Memo;
+
+class MemoController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $query = Memo::query();
+
+        if ($search = $request->input('search')) {
+            $query->where(function($q) use ($search) {
+                $q->where('title', 'LIKE', "%{$search}%")
+                ->orWhere('content', 'LIKE', "%{$search}%");
```

Comment:
以下のようにしたら検索できますかね？？
viewのbladeファイル以外では`{}`は使えず、今まで通り `.` で文字列連結する必要があります！

```suggestion
                $q->where('title', 'LIKE', "%" . $search . "%")
                ->orWhere('content', 'LIKE', "%" . $search . "%");
```

Reply:
修正しましたが、検索出来ないです、、
この修正をするにあたってほかファイルの修正もする必要があるのかと検索かけましたが分からずでした、、